cdap-explore/src/main/java/co/cask/cdap/explore/service/hive/BaseHiveExploreService.java

```diff
@@ -667,11 +676,18 @@ public QueryHandle execute(Id.Namespace namespace, String statement) throws Expl
     startAndWait();
 
     try {
-      Map<String, String> sessionConf = startSession();
+      Map<String, String> sessionConf = startSession(namespace);
       // It looks like the username and password below is not used when security is disabled in Hive Server2.
       SessionHandle sessionHandle = cliService.openSession("", "", sessionConf);
       try {
         String database = getHiveDatabase(namespace.getId());
+        String schedulerQueue = getNamespaceResolvedSchedulerQueue(namespace);
+        if (schedulerQueue != null) {
+          sessionConf.put(Constants.MapReduce.MAP_REDUCE_JOB_QUEUE_NAME, schedulerQueue);
```

Comment:
No need to add job queue name to session conf again, as startSession already does it, right?